Taskfile.yml

```diff
@@ -48,6 +47,10 @@ tasks:
       - ./*.go
       - ./**/*.go
 
+  dev-build:
+    deps: [generate, build]
+    desc: "Developer build from scratch (regenerates internal resources)"
+
```

Comment:
I think the better fix would be to check in ``scripts/generate.sh`` to check if the ``capnp`` executable is available. No need for an additional build target then.

Reply:
Did as you suggested in a118c47 and reverted Taskfile back with  27189de